server/views/components/alertsList.njk

```diff
@@ -13,7 +13,7 @@
         {% endif %}
         {% if 'XA' in alertCodes %}
             <li class='alert-status alert-status--arsonist'>
-                <img src='/assets/images/arson_icon.png' alt='Arson'>
```

Comment:
Cypress gave me accessiblity warnings about alt being same / similar to text so advice was to leave alt blank.